.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,33 @@
+name: Feature request
+description: Suggest an idea or new component for this project.
+labels: ["Type: Feature Request"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        * We welcome any ideas or feature requests! It’s helpful if you can explain exactly why the feature would be useful.
+        * There are usually some outstanding feature requests in the [existing issues list](https://github.com/espressif/esp-usb/labels/Type%3A%20Feature%20Request), feel free to add comments to them.
```

Comment:
Here as well, link to `esp-usb` github

Reply:
I removed this template as I don't see it can be helpful.
Just based on the rate how often we get such feature-requests in espressif/tinyusb.